blogposts/2021/release-3.30.md

```diff
@@ -0,0 +1,204 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/22570
+    category:
+  - description: A new bulk operation to close many changesets at once has been added to Batch Changes.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22547
+    category: Batch Changes
+  - description: Backend Code Insights will aggregate viewable repositories based on the authenticated user.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22471
+    category: Repositories
+  - description: Added support for highlighting .frugal files as Thrift syntax.
+    url:
+    category:
+  - description: "Added `file:contains.content(regexp)` predicate, which filters only to files that contain matches of the given pattern."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22666
+    category:
+  - description: "Repository syncing is now done in streaming mode by default. Customers with many repositories should notice code host updates much faster, with repo-updater consuming less memory. Using the previous batch mode can be done by setting the `ENABLE_STREAMING_REPOS_SYNCER` environment variable to `false` in `repo-updater`. That environment variable will be deleted in the next release."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22756
+    category: Batch Changes
+  - description: Enabled the ability to query Batch Changes changesets, changesets stats, and file diff stats for an individual repository via the Sourcegraph GraphQL API.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22744/
+    category: Batch Changes
+  - description: "Added \"Groovy\" to the initial `lang:` filter suggestions in the search bar."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22755
+    category: Search
+  - description: "The `lang:` filter suggestions now show all supported, matching languages as the user types a language name."
+    url: https://github.com/sourcegraph/sourcegraph/pull/22765
+    category:
+  - description: Code Insights can now be grouped into dashboards.
+    url: https://github.com/sourcegraph/sourcegraph/issues/22215
+    category:
+  - description: "Batch Changes changesets can now be [published from the Sourcegraph UI](https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui)."
+    url: https://github.com/sourcegraph/sourcegraph/issues/18277
+    category: Batch Changes
+  - description: The repository page now has a new button to view batch change changesets created in that specific repository, with a badge indicating how many changesets are currently open.
+    url: https://github.com/sourcegraph/sourcegraph/pull/22804
+    category: Batch Changes
+  - description: "Experimental: Search-based code insights can run over all repositories on the instance. To enable, use the feature flag `\"experimentalFeatures\": { \"codeInsightsAllRepos\": true }`."
+    url: https://github.com/sourcegraph/sourcegraph/issues/22759
+    category: Search
```

Comment:
@Joelkw - should this be Code Insights?

Reply:
@abschu yep! I think it's okay if you feel free to change the categories without requiring approval – it seems like they're being attached based on wording -- note they don't appear in the changelog itself so I assume the script is trying to match on simple words like "search" in the description.